buildout.cfg

```diff
@@ -2,12 +2,13 @@
 extends =
     https://zopefoundation.github.io/Zope/releases/4.x/versions.cfg
 develop = .
-parts = 
-    interpreter 
-    test 
+parts =
+    interpreter
+    test
 
 [versions]
 Products.ZCatalog =
+RestrictedPython = >= 5.1
```

Comment:
Remove this pin, you're already pinning in setup.py

Reply:
Without this pin the tests fail. It actually removes the constraint set in the `versions.cfg` file used via `extends` as it requires `5.0` which is not compatible with the requirement from `setup.py`.

We can remove it when we switch the versions used in `extends` to Zope 5.